crates/proposer-elfs/build.rs

```diff
@@ -0,0 +1,12 @@
+fn main() {
+    prover_elf_utils::ElfInfo::new("elf_info.rs")
```

Comment:
It is a bit confusing that both `crates/prover-elf-utils/src/elf_info.rs` and this generated file have the same name.